client/src/components/registration/RegistrationForm.js

```diff
@@ -85,6 +101,20 @@ const RegistrationForm = () => {
       ...userPayload,
       [event.currentTarget.name]: event.currentTarget.value,
     });
+    if (event.currentTarget.name === "admin") {
+      if (event.currentTarget.checked) {
+        setUserPayload({
+          ...userPayload,
+          [event.currentTarget.name]: true
+        });
+      }
+      else {
+        setUserPayload({
+          ...userPayload,
+          [event.currentTarget.name]: false,
+        });
+      }
+    }
```

Comment:
first: we dont want to let just any person designate themselves as admin right? so there really shouldnt be any mention of admin on this form or the backend endpoint

second: I dont actually see an option for designating as admin, so I dont even think this admin-related-code is being used